x-pack/plugin/core/src/main/resources/logs-default-policy.json

```diff
@@ -0,0 +1,13 @@
+{
+  "phases": {
+    "hot": {
+      "min_age": "0ms",
```

Comment:
is this `min_age` required? What is the effect of it?

Reply:
It's not required, and 0 is the default so this can be removed